### PR TITLE
Set default values

### PR DESCRIPTION
**Pull request by masterT**
- set default values
- extract the code that format the query params in a new function
‘formatQueryParams’
- add test for default values

Fix issue #20

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -61,12 +55,12 @@ var generateQueryString = function(query, method, credentials) {
...
         defaultParams['IncludeReviewsSummary'] = 'True';
         defaultParams['TruncateReviewsAt'] = '1000';
-        defaultParams['VariationPage'] = 'All';
-        defaultParams['MaximumPrice'] = '';
+        defaultParams['VariationPage'] = 'All';        
 
-        for (param in query) {
-            var capitalized = capitalize(param);
-            params[capitalized] = query[param] || defaultParams[capitalized];
+        for (param in defaultParams) {
```
Comment (t3chnoboy) [inline lib/utils.js]: I see this pattern in couple of places. Can you extract it into a function?

Comment (masterT) [inline lib/utils.js]: of course

Comment (t3chnoboy) [inline lib/utils.js]: Thanks! Otherwise looks good!

**Comment by t3chnoboy** [comment]
> All checks have passed
whoa. PR checker works now!

**Comment by masterT** [comment]
Now I don't see the point of having a variable **defaultParams**. I could refactor a little more and change this:
```js
// Default
defaultParams['SearchIndex'] = 'All';
defaultParams['Condition'] = 'All';
defaultParams['ResponseGroup'] = 'ItemAttributes';
defaultParams['Keywords'] = '';
defaultParams['ItemPage'] = '1';

params = setDefaultParams(params, defaultParams);
```

to that:
```js
```

**Comment by t3chnoboy** [comment]
sweet! Thanks a lot for what you are doing to this project!

**Comment by masterT** [comment]
You are welcomed! :blush:

**Comment by masterT** [comment]
I think everything is good now.

**Comment by masterT** [comment]
@t3chnoboy  I will merge/dump version/publish good ?

**Comment by t3chnoboy** [comment]
yep! shipit :ship:

**Comment by t3chnoboy** [comment]
that was fast :car:

**Comment by masterT** [comment]
:information_desk_person:

**Pull request merged**